decred/tests/unit/dcr/test_vsp_unit.py

```diff
@@ -142,9 +142,11 @@ def test_pool_stats():
 now = int(time.time())
 votingServiceProvider = {
     "url": "https://www.dcrstakedinner.com",
-    "apiKey": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE1Nzc0MzM0NDIsIm"
-    "lzcyI6Imh0dHBzOi8vd3d3LmRjcnN0YWtlZGlubmVyLmNvbSIsImxvZ2dlZEluQXMiOjQ2fQ."
-    "PEb000_TjQuBYxjRdh-VOaXMdV2GUw3_ZyIyp_tfpFE",
+    "apiKey": (
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE1Nzc0MzM0NDIsIm"
+        "lzcyI6Imh0dHBzOi8vd3d3LmRjcnN0YWtlZGlubmVyLmNvbSIsImxvZ2dlZEluQ"
+        "XMiOjQ2fQ.PEb000_TjQuBYxjRdh-VOaXMdV2GUw3_ZyIyp_tfpFE"
+    ),
```

Comment:
Thank you, I realized this was wrong after the fact but then forgot to change it.